warehouse/models/intermediate/gtfs/_int_gtfs.yaml

```diff
@@ -620,8 +619,8 @@ models:
             - trip_route_id
             - trip_direction_id
             - schedule_base64_url
-          # TODO: uncomment once this is passing in prod
-          # where: '__rt_sampled__'
+          # we have a very small number of historical failures on this uniqueness on days where a given feed's time zone changed
```

Comment:
just to confirm, this test can theoretically fail on historical data, correct?

Reply:
yes